config/schemas/config.yaml

```
@@ -4061,6 +4061,131 @@ properties:
           customTemplate:
             title: Slack Custom Template
             type: string
+      runbookUrls:
+        title: Alert runbooks
+        description: |-
+          Configure runbooks for alerts
+
+          Runbooks can be configured on an alert group level or per individual alert
+        type: object
+        additionalProperties: false
+        $defs:
+          upstreamRunbook:
+            title: Runbooks for alerts
+            description: |-
+              Example:
+
+              group: link-to-alert-group-runbook
+              AlertName: link-to-specific-alert-runbook
+
+              Uses upstream runbooks by default
+
+              https://runbooks.prometheus-operator.dev/runbooks/
+            type: object
+            additionalProperties:
+              type: string
```

Minor: If they're all URLs then setting a format may do some validation of that. 

```suggestion
              type: string
              format: uri
```

---

The default value if there's no upstream runbook is "Missing runbook", I think it could be allowed to set a non-uri string there if you want to write some other information